zzz_generated/app-common/app-common/zap-generated/attributes/Accessors.h

```diff
@@ -1963,6 +1963,37 @@ EmberAfStatus Set(chip::EndpointId endpoint, uint16_t value);
 } // namespace Attributes
 } // namespace BooleanState
 
+namespace ClientMonitoring {
+namespace Attributes {
+
+namespace IdleModeInterval {
+EmberAfStatus Get(chip::EndpointId endpoint, uint32_t * value); // int32u
+EmberAfStatus Set(chip::EndpointId endpoint, uint32_t value);
+} // namespace IdleModeInterval
+
+namespace ActiveModeInterval {
+EmberAfStatus Get(chip::EndpointId endpoint, uint32_t * value); // int32u
+EmberAfStatus Set(chip::EndpointId endpoint, uint32_t value);
+} // namespace ActiveModeInterval
+
+namespace ActiveModeThreshold {
+EmberAfStatus Get(chip::EndpointId endpoint, uint16_t * value); // int16u
+EmberAfStatus Set(chip::EndpointId endpoint, uint16_t value);
+} // namespace ActiveModeThreshold
+
```

Comment:
According to the .xml definition `RegisteredClients` list is missing here. Is it expected and you plan to add it later in some followup PR?

Reply:
Based on my understanding, since `RegisteredClients` is an Array, it does not exist in the Accessors generation. The implementation of the attribute will be added and verified in subsequent PRs.